send/pithia_portal.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+import os
+import sys
+import subprocess
+import send_lib
+import requests
+
+SERVICE_NAME = "pithia_portal"
+os.environ["SERVICE_NAME"] = SERVICE_NAME
+
+facility = sys.argv[1]
+destination = sys.argv[2]
+destination_type = sys.argv[3]
+auth_token = None
+
+auth = send_lib.get_auth_credentials(SERVICE_NAME, destination)
+url = send_lib.get_custom_config_properties(SERVICE_NAME, destination, ["url"])[0]
+data = {
+	"username": auth[0],
+	"password": auth[1]
+}
+response = requests.post(url, data)
+
+if response.status_code == 200:
+	response = response.json()
+	if "token" not in response:
+		send_lib.die_with_error("Token missing in response")
```

Comment:
`Token missing in response from token server` - so it can't be confused with an error from the actual service endpoint.